packages/react/src/components/Table/TableBody/RowActionsCell/RowActionsError.jsx

```diff
@@ -39,7 +39,7 @@ const RowActionsError = ({
     return null;
   }
   const { title, message, learnMoreURL } = rowActionsError;
-  return rowActionsError ? (
```

Comment:
Is this no longer needed?

Reply:
correct, at some point this was added right before
```
  if (!rowActionsError) {
    return null;
  }
```